soci/artifacts.go

```diff
@@ -63,6 +63,7 @@ var (
 	bucketKeyCreatedAt      = []byte("created_at")
 
 	artifactsDbName = "artifacts.db"
+	ArtifactsDbPath = path.Join(config.SociSnapshotterRootPath, artifactsDbName)
```

Comment:
Not really a blocker for this PR since some vars already exist.

I don't think we should have export vars. That means these vars can be accidentally modified by anyone. We should change `artifactsDbName` and `ArtifactEntryTypeIndex` to const; `CreatedAt` to `createAt` and create getter func if it's needed externally.

For `ArtifactsDbPath` I assume it's not intended to be modified after init? (e.g. always = `path.Join(config.SociSnapshotterRootPath, artifactsDbName)`) If so we should change it to `artifactsDbPath` and create a getter func, or make it a func directly, something like,

```golang
func ArtifactsDbPath() {
    return path.Join(config.SociSnapshotterRootPath, artifactsDbName)
}
```

Reply:
thanks, i'll apply the suggestions in the next pr